pkg/cli/admin/upgrade/status/examples/4.14.1-degraded.detailed-output

```diff
@@ -1,6 +1,7 @@
 = Control Plane =
 Assessment:      Stalled
 Target Version:  4.14.1 (from 4.14.0-rc.3)
+Waiting On:      machine-config
```

Comment:
"waiting on" is good in isolation, but we're using "waiting" verb for "not updating yet" in `Completion:` line  which I think would be confusing

Reply:
Changed to "Updating" to be aligned with the Completion line.